### ISSUE TITLE
TFH Dashboard Down for Maintenance

### ISSUE THREAD

**Issue by Accio1** [theforumhelpers/theforumhelpers.github.io#297]
TFH Dashboard is down for maintenance

Reason(s):
- Creating a testing database for local development that will not impact the production database.
- Rewriting some parts to make it easier for others to contribute.
- Moving some things around in my own local filesystem to make it easier for me to develop this going forward.

Impact:
- Visiting any part of the dashboard will result in a maintenance page.
- Application status and codeword is not displayed on the main site.

Estimated return of service:
1:00 PM EST on August 19th (or earlier)

**Comment by gosoccerboy5** [comment]
when will it be open-sourced, if at all?

**Comment by Accio1** [comment]
@gosoccerboy5 eventually. I don't have a set date yet but it will probably be some time after the next major update is released, which will include curators being able to sign in as well.

**Comment by gosoccerboy5** [comment]
alright. I would love to see what goes on behind the scenes.

**Comment by Accio1** [comment]
It is a big mishmash with little to no documentation (part of the reason why I haven't released it - it is very unorganized). The single dependency it has took me around a day to set up on my Raspberry Pi, which is where I have done the majority of work on it.

**Comment by gosoccerboy5** [comment]
wait, so the server is a raspberry pi?

**Comment by Accio1** [comment]
No. I coded it on a Raspberry Pi because I like Linux better than Windows for coding.

**Comment by gosoccerboy5** [comment]
@Accio1 don't mean to be bothersome but would it be possible to post at least a mirror of TFH Dashboard's source code? maybe in github gists?

**Comment by gosoccerboy5** [comment]
Ok, neat.

**Comment by penguinmoose** [comment]
> > @Accio1 don't mean to be bothersome but would it be possible to post at least a mirror of TFH Dashboard's source code? maybe in github gists?
> 
> @gosoccerboy5 This is coming eventually. My pessimistic timeline is by the end of the year, and my optimistic timeline is by the end of September, so it will probably be somewhere in between.

Good to know
